addons/udes_stock/views/stock_picking.xml

```diff
@@ -289,4 +289,23 @@
         </field>
     </record>
 
+    <record id="view_picking_warning" model="ir.ui.view">
+        <field name="name">stock.inventory.adjust.reserved.view.form</field>
+        <field name="model">stock.inventory</field>
+        <field name="priority">20</field>
+        <field name="arch" type="xml">
+            <form string="Warning">
+                <group>
+                    <p>
+                        Do you wish to continue?
```

Comment:
Super nit: Could the `<p>` content be one line?

Reply:
Fixed